backend/src/main/java/com/speakfluid/backend/entities/ButtonStep.java

```diff
@@ -11,32 +11,36 @@
  * the usefulness of a button for this step in the conversational flow. It implements the TalkStep abstract class
  * and inherits all of its attributes and methods.
  * @author Aurora Zhang
- * @version 1.0
+ * @version 3.0
  * @since November 16th 2022
  */
 
 public class ButtonStep extends TalkStep{
 
     private double scoreAccumulator;
-    private final double maxScore = 22.0;
+    private final double maxScore = ScoreStandards.standardStepClass;
     private final String stepName = "Button";
 
 
     public ButtonStep(){
         this.scoreAccumulator = 0.0;
     }
     private final List<Map<String, Double>> chatbotKeywordsScoreMap = Arrays.asList(
```

Comment:
Is there a better way of accomplishing this rather than having all of these similar lines of code? It's not as easy to read the code